modules/survey_accounts/test/survey_accountsTest.php

```diff
@@ -225,14 +288,52 @@ function testSurveyAccountsAddSurvey()
         $this->safeFindElement(
             WebDriverBy::Name("PSCID")
         )->sendKeys("8889");
+        $this->safeFindElement(
+            WebDriverBy::Name("VL")
+        )->sendKeys($invalidVisitLabel);
```

Comment:
This section of the test also causes an integration test error. Since the visit label selection is a drop-down menu with the options `V1-V6`, trying to pass an empty string here causes this error when the test is run:
`Facebook\WebDriver\Exception\NoSuchElementException: Cannot locate option with value: ' '`

I suggest that this section of the test be removed.